email/getConfigReceiver.js

```diff
@@ -0,0 +1,15 @@
+const frappe = require('frappejs');
```

Comment:
You can commonify the code to get receiver and sender account config.... you can pass filters accordingly

Reply:
Done . 
i didn't use filters as we need to work on another issue regarding `defaultIncoming` and `defaultOutgoing` , so that incorporating those changes would take less time IMO .